### PR TITLE
Beta switcher - initial implementation

### PR DESCRIPTION
**Pull request by fhlavac**
https://issues.redhat.com/browse/RHCLOUD-6660

- added beta-switcher + notification on the switch
- updated snapshot

@john-dupuy

### CODE REVIEW

**Review by PanSpagetka — APPROVED** [review]

File: src/js/App/Header/Tools.js [hunk 1]
```diff
@@ -32,20 +47,37 @@ const Tools = () => {
...
-      variant="plain"
-      aria-label="Go to settings"
-      ouiaId="chrome-settings"
-      className="ins-c-toolbar__button-settings"
-      href={settingsPath}
-      component="a"
-    >
-      <CogIcon />
-    </Button>
+    <ToolbarToggle
```
Comment (john-dupuy) [inline src/js/App/Header/Tools.js:68]: Can we add an ouiaId here? It can be the same as before `chrome-settings`.

**Review by john-dupuy — CHANGES_REQUESTED** [review]

File: src/js/App/Header/Tools.js (hunk 1 above)
Comment (fhlavac) [reply src/js/App/Header/Tools.js:68]: just added it, sorry :-)

Comment (john-dupuy) [reply src/js/App/Header/Tools.js:68]: Thanks!

**Review by john-dupuy — APPROVED** [review]
LGTM, @karelhala would it be possible to leave put this only in CI-beta before we merge it into QA-beta? I want to write the automation so that tests won't fail once this gets into QA-beta.

**Comment by karelhala** [comment]
@john-dupuy yeah, no problem.

**Comment by bclarhk** [comment]
The verbiage did not reflect what was in the mocks, so I got our UX content specialists to look the whole thing over before I requested any changes. The mocks have been updated with the following:

- dropdown menu in stable: Use the beta release
- toast after entering beta: You're now using the beta release.
- dropdown menu in beta: Stop using the beta release
- toast after leaving beta: You're no longer using the beta release.

**Comment by fhlavac** [comment]
@bclarhk thanks for catching that, I was trying to follow the mocks, I may have missed something. Now it should be fixed according to your request. I just want to make sure... the notification should be visible until the user clicks the close button? Or should it disappear automatically after a certain time? Thank you :)

**Comment by ryelo** [comment]
@fhlavac I believe if we're following the _new_ spec for the notifications:
* You should be able to close it with the `x`
* If you do not, the toast should close by itself after 5 seconds

**Comment by fhlavac** [comment]
@ryelo that's how it works now, thank you :)

File: src/js/App/Header/HeaderAlert.js [hunk 2]
```diff
@@ -0,0 +1,56 @@
...
+  const [alertVisible, setAlertVisible] = useState(true);
+  const [timer, setTimer] = useState(null);
+
+  useEffect(() => {
+    dismissable || createTimer();
+  }, []);
+
+  useEffect(() => {
+    return () => clearTimeout(timer);
+  }, []);
```
Comment (karelhala) [inline src/js/App/Header/HeaderAlert.js:17]: You can merge these two together.
```suggestion
  useEffect(() => {
    dismissable || createTimer();
    return () => clearTimeout(timer);
  }, []);
```

**Review by Hyperkid123 — APPROVED** [review]

**Pull request merged**